docs/going-into-production.rst

```diff
@@ -0,0 +1,357 @@
+.. _going-into-production:
+
+=====================
+Going into production
+=====================
+
+Running CrateDB in different environments requires different approaches. This
+document outlines the basics you need to consider when going into production.
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+
+
+.. _prod-bootstrapping:
+
+Configure bootstrapping
+=======================
+
+The process of forming a cluster is known as *bootstrapping*. Consult the
+how-to guide on :ref:`CrateDB multi-node setups <multi_node_setup>` for an
+overview of the two different ways to bootstrap a cluster.
+
+If you have been using CrateDB for development on your local machine, there is
+a good chance you have been using :ref:`single host auto-bootstrapping
+<auto-bootstrapping>`.
+
+For improved performance and resiliency, you should run production CrateDB
```

Comment:
Should we include an `we recommend to use at least three nodes`?
I think we should mention this here, because that's what we recommend and there is no dedicated place for this information yet. Of course it is mentioned in the multi-node setup, but to get there you need to know you want to do this.